src/pages/purchase-orders/edit/components/Actions.tsx

```diff
@@ -170,6 +172,7 @@ export function useActions() {
             .finally(() => invalidateCache(po.id)),
         hideIf: po.archived_at > 0,
         icon: <Icon element={MdArchive} />,
+        showDivider: true,
```

Comment:
Please stick to this way of using dividers:

https://github.com/invoiceninja/ui/blob/main/src/pages/quotes/common/hooks.tsx#L338

Reply:
Sure, the code has been refactored with the current convenction of handling more_actions.